rpi-update

```diff
@@ -370,7 +380,8 @@ function finalise {
 function do_backup {
 	if [[ ${SKIP_BACKUP} -eq 0 ]]; then
 		echo " *** Backing up files (this will take a few minutes)"
-		local OLD_FW_PATH=${FW_PATH}.bak
+		local OLD_FW_PATH="${BOOT_PATH}.bak${FW_SUBDIR}"
```

Comment:
Do we want separate backups for each os_prefix?
I'm a little concerned these are quite big and rarely used.
A single backup should mean you can get back to a bootable system, and from there update to a specific system.

Reply:
That's up to you. Either way, this line needs to change, the difference being whether or not you include ${FW_SUBDIR} in OLD_FW_PATH.

Reply:
I think I'd prefer it just became `${BOOT_PATH}.bak`

Reply:
It now uses `${BOOT_PATH}.bak`.